ci/build_wheel.sh

```diff
@@ -38,6 +38,14 @@ if [[ ${package_name} != "libraft" ]]; then
     )
 fi
 
+RAPIDS_CUDA_MAJOR="${RAPIDS_CUDA_VERSION%%.*}"
+if [[ ${RAPIDS_CUDA_MAJOR} == "12" ]]; then
```

Comment:
Is there a reason we do this only for CUDA 12?

Reply:
Yes, there's no NCCL wheel distributions available for CUDA 11 since April 2024 and even then, no arm64 binaries. I think we'll have to continue to vendor NCCL binaries with CUDA 11.

Reply:
Great. That is fine! I just didn't know the constraints off the top of my head.